index.html

```diff
@@ -109,16 +120,64 @@ <h2>2. Hogyan zajlik egy webdesign projekt?</h2>
             hogy minden jól működik. Az általános tesztelés mellett a webdesigner teszteli a weboldalt különböző böngészőkben és készülékeken is. Amennyiben talál 
             valamilyen hibát, azokat kijavítja és optimalizálja a honlapot.</span>
         </p>
+    </div>
+
+    <div class="picture">
+        <h2>A CSS kód</h2>  
         <hr>
+        <p>A kép címe: CSS kód</p>  
+        <img src="src/assets/code.png" alt="kep">
     </div>
+    
+    <br>
+
+    <div class="formOne"> <!-- Error: Bad value for attribute action on element form: Must be non-empty - ezt a validátorok hibának jelzik, de Csabi azt kérte maradjon üres! - Rendes mezőellenőrzések, majd javascriptben kellenek késöbb -->
+        <form action="">
+            <h2 id="headerForm">Cikk küldése ismerősnek</h2>
+            <div>
+                <label for="nevImput">Név:</label>
+                <input type="text" name="nev" id="nevImput" maxlength="30" placeholder="Teszt Elek" required> <!--kötelező mező-->
+            </div>
+            <div>
+                <label for="telInput">Telefonszám:</label>
+                <input type="tel" name="telefonszam" id="telInput" maxlength="12" placeholder="+36703002020">
+            </div>
+            <div>
+                <label for="emailInput">Email cím:</label>
+                <input type="email" name="email" id="emailInput" maxlength="100" placeholder="teszt@teszt.hu" required> <!--kötelező mező-->
+            </div>
+            <div>
+                <label for="koraInput">Kora:</label>
```

Comment:
Vegyesen használod a nyelveket a változónevekben, szintén nem túl jövőálló (pl. fél év múlva ez a kód már óriási lesz és meg akarod keresni gyorsan a kor beviteli mező formázását a css-ben, mit írsz? ageInput, koraInput?)

(amúgy igen, órán én is összevissza csináltam, elrettentő példának! :)